### PR TITLE
Single celery

### PR DESCRIPTION
**Pull request by maparent**

### CODE REVIEW

File: assembl/lib/model_watcher.py [hunk 1]
```diff
@@ -67,45 +72,112 @@ def processAccountModified(self, id):
         pass
 
 
-class ModelEventWatcherPrinter(object):
+class BaseModelEventWatcher(object):
+    """A dummy :py:class:`IModelEventWatcher` for testing purposes"""
+    interface.implements(IModelEventWatcher)
```
Comment (vincentfretin) [inline assembl/lib/model_watcher.py]: This is not compatible python 3. You should use `@implementer(IModelEventWatcher)` on the class.

**Comment by ayyazdaniaryan** [comment]
@vincentfretin Yes, you would. There is a process (and I don't believe it's documented anywhere) were you delete ALL `NotificationSubscriptions` on the server. And you remove existing Notification objects in case the status is some is still pending. This process was done only once for dev-staging server, and from which, all local databases are based on.

**Comment by vincentfretin** [comment]
We sometime take a real prod dump, I did it once to fix an issue we had only on prod to work on it locally.
We definitely should have a script we can execute to remove NotificationSubscriptions and pending notifications like you said.

**Comment by maparent** [comment]
Good point on prod dump. It has not been an issue on our (Mac) machines because we don't have an active sendmail, but it would be an issue on a linux dev machine. Will look into it.

**Comment by maparent** [comment]
Ok. Added a parameter to develop.rc so the notify task will obsolete the notification instead of sending it. I think that's the best solution.

**Comment by maparent** [comment]
Is that a merge approval?

**Pull request merged**